01.py

```diff
@@ -56,4 +56,9 @@
 rdd = sc.newAPIHadoopRDD('io.warp10.hadoop.Warp10InputFormat', 'org.apache.hadoop.io.Text', 'org.apache.hadoop.io.BytesWritable', conf=conf)
 df = rdd.toDF()
 
-print df.take(10)
+schemaTreeString = df._jdf.schema().treeString()
```

Comment:
This breaks the conciseness of the previous script, can you make it more compact?